examples/app.py

```diff
@@ -35,35 +51,96 @@
 
 You can find the database in `examples/app.db`.
 
-Run the development server:
+
+3. Transform the file `authority.xml` from MARC21 to a json file:
+
+.. code-block:: console
+
+    $ dojson do -i authority.xml -l marcxml marc21 > records.json
```

Comment:
Please use the file from `invenio-records`.

--
see https://github.com/inveniosoftware/invenio/blob/master/scripts/populate-instance.sh#L115-L119